staff/lucas-wloch/challenges/tic-tac-toe-app/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tic Tac Toe</title>
+    <link rel="stylesheet" href="styles.css">
+    <script src="main2.js" defer></script>
```

Comment:
you have to change this, please put it under </body>

Reply:
Thanks